sklearn/metrics/classification.py

```diff
@@ -2038,3 +2038,73 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
     y_true = np.array(y_true == pos_label, int)
     y_true = _check_binary_probabilistic_predictions(y_true, y_prob)
     return np.average((y_true - y_prob) ** 2, weights=sample_weight)
+
+
+def fall_out(y_true, y_pred, labels=None, sample_weight=None):
+    """Parameters
+       ----------
+       y_true : array, shape = [n_samples]
+           True target, consisting of integers of two values. The positive label
+           must be greater than the negative label.
+
+       pred_decision : array, shape = [n_samples] or [n_samples, n_classes]
+           Predicted decisions, as output by decision_function (floats).
+
+       labels : array, optional, default None
+           Contains all the labels for the problem. Used in multiclass hinge loss.
+
+       sample_weight : array-like of shape = [n_samples], optional
+           Sample weights.
+
+       Returns
+       -------
+       loss : float"""
+    CM = confusion_matrix(y_true, y_pred, labels=labels, sample_weight=sample_weight)
+    return CM[0][1]
+
+
+def Miss_rate(y_true, y_pred, labels=None, sample_weight=None):
+    """Parameters
+    ----------
+    y_true : array, shape = [n_samples]
+        True target, consisting of integers of two values. The positive label
+        must be greater than the negative label.
+
+    pred_decision : array, shape = [n_samples] or [n_samples, n_classes]
+        Predicted decisions, as output by decision_function (floats).
+
+    labels : array, optional, default None
+        Contains all the labels for the problem. Used in multiclass hinge loss.
+
+    sample_weight : array-like of shape = [n_samples], optional
+        Sample weights.
+
+    Returns
+    -------
+    loss : float"""
+    CM = confusion_matrix(y_true, y_pred, labels=labels, sample_weight=sample_weight)
+    return CM[1][0]
+
+
+def specificity(y_true, y_pred, labels=None, sample_weight=None):
+    """Parameters
+    ----------
+    y_true : array, shape = [n_samples]
```

Comment:
It would be good if you write a little comment about this new feature

Reply:
thanks for the feedback i will just add